docs/api-reference.md

```
@@ -495,3 +528,53 @@ open doc/index.html
+Benefits:
+
+- IDE autocomplete with method signatures and parameter hints.
+- Static type checking with tools such as
+  [Steep](https://github.com/soutaro/steep) or
+  [TypeProf](https://github.com/ruby/typeprof).
+- Machine-readable API documentation.
+- Safer refactoring across Shakapacker API calls.
+
+RBS signatures are in the `sig/` directory and included with the gem:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)